<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyInforDialog</class>
 <widget class="QDialog" name="MyInforDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="WorkTelEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>56</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>性别:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>婚姻状况:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>籍贯:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="StateEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>工作岗位:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>56</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>民族:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>政治面貌:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>员工号:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>办公电话:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>到岗日期:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>备注:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="TelEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="BackgroundEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>56</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>姓名:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>出生日期:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>文化程度:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ReturnButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QLineEdit" name="NameEdit_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="BrithdayEdit_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="CultureEdit_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="PlaceEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>职务:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>61</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <widget class="QLineEdit" name="SexEdit_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="MerryEdit_2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="WhereEdit_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="PostEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="OtherEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="ReachEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>身份证号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_31">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>户口所在地:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PersonNumberEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="WorkNumEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_32">
      <property name="text">
       <string>填表人: admin</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_33">
      <property name="text">
       <string>填表日期:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_34">
      <property name="text">
       <string>2010-01-01</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_35">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>合同有效期:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SituationEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_36">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>手机号码:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_37">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>员工状态:</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>81</width>
     <height>111</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>79</width>
      <height>109</height>
     </rect>
    </property>
    <widget class="QLabel" name="photolabel_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>81</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="photolabel_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>81</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLineEdit" name="NationEdit">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PeriodEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-7</x>
     <y>0</y>
     <width>621</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/background_meitu_2.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>WorkTelEdit</zorder>
  <zorder>layoutWidget_6</zorder>
  <zorder>StateEdit</zorder>
  <zorder>label_11</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>TelEdit</zorder>
  <zorder>BackgroundEdit</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>ReturnButton</zorder>
  <zorder>layoutWidget_5</zorder>
  <zorder>PlaceEdit</zorder>
  <zorder>label_29</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>PostEdit</zorder>
  <zorder>OtherEdit</zorder>
  <zorder>ReachEdit</zorder>
  <zorder>label_30</zorder>
  <zorder>label_31</zorder>
  <zorder>PersonNumberEdit</zorder>
  <zorder>WorkNumEdit</zorder>
  <zorder>layoutWidget_4</zorder>
  <zorder>label_35</zorder>
  <zorder>SituationEdit</zorder>
  <zorder>label_36</zorder>
  <zorder>label_37</zorder>
  <zorder>scrollArea</zorder>
  <zorder>NationEdit</zorder>
  <zorder>PeriodEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
